packages/tools/devtools/devtools-view/src/components/Menu.tsx

```diff
@@ -390,6 +553,13 @@ export interface MenuProps {
 	 * The set of Containers to offer as selection options.
 	 */
 	containers?: ContainerKey[];
+
+	/**
+	 * Callback function when a container should be removed from the list.
```

Comment:
Nit (to be a bit more explicit): what do you think about this?
```suggestion
	 * Callback function when a container should be deregistered from the devtools.
```